components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -18069,6 +18210,52 @@ public Set<SubscribedAPI> getPaginatedSubscribedAPIsByApplication(Application ap
 
     }
 
+    public List<APIEndpointInfo> getAPIEndpoints(String uuid) throws APIManagementException {
+        List<APIEndpointInfo> apiEndpoints = null;
+        String sql;
+        APIRevision apiRevision = checkAPIUUIDIsARevisionUUID(uuid);
+        String currentApiUuid;
+        boolean isRevision = false;
+        if (apiRevision != null && apiRevision.getApiUUID() != null) {
+            currentApiUuid = apiRevision.getApiUUID();
+            sql = SQLConstants.APIEndpointsSQLConstants.GET_ALL_API_ENDPOINTS_BY_API_UUID_REVISION_SQL;
+            isRevision = true;
+        } else {
+            sql = SQLConstants.APIEndpointsSQLConstants.GET_ALL_API_ENDPOINTS_BY_API_UUID;
+            currentApiUuid = uuid;
+        }
+
+        try (Connection conn = APIMgtDBUtil.getConnection()) {
+            int apiId = getAPIID(currentApiUuid, conn);
+            PreparedStatement ps = conn.prepareStatement(sql);
+            ps.setInt(1, apiId);
+            ps.setString(2, APIConstants.APIEndpoint.ENDPOINT_NONE_NAME);
+            if (isRevision) {
+                ps.setString(3, apiRevision.getRevisionUUID());
+            }
+            try (ResultSet rs = ps.executeQuery()) {
+                if (rs != null) {
+                    apiEndpoints = new ArrayList<>();
+                    while (rs.next()) {
```

Comment:
Let's make these strings constants - "ENDPOINT_UUID", "ENDPOINT_NAME", "ENDPOINT_TYPE", "ENDPOINT_CONFIG", "ORGANIZATION"

Reply:
All methods in DB fetching column labels are instantly defined. They do not use constants before for column labels that's why I also define them instantly.